events/equeue/equeue_posix.c

```diff
@@ -25,33 +25,39 @@
 
 
 // Tick operations
-unsigned equeue_tick(void) {
+unsigned equeue_tick(void)
+{
     struct timeval tv;
     gettimeofday(&tv, 0);
-    return (unsigned)(tv.tv_sec*1000 + tv.tv_usec/1000);
+    return (unsigned)(tv.tv_sec * 1000 + tv.tv_usec / 1000);
```

Comment:
This code just go harder to understand (where's the precedence?). Is it possible to make astyle be more lax around expression spacing?

Reply:
Yes there is a rule. To answer precedence - use braces.
we do spaces around operators

Reply:
IMO this is more harmful than helpful (what do standard spaces around operators help with?) but fair point on using braces.